content/blog/2017-10-27-GMT5-Windows-Warning.md

```diff
@@ -0,0 +1,27 @@
+---
+title: GMT 5 在 Windows 下提示无法创建用户目录的解决办法
+date: 2017-10-27
+categories:
+    - GMT技巧
+authors:
+    - 赵志远
+slug: GMT5-Windows-Warning
```

Comment:
Please change slug to "user-directory-warning", and filename to "2017-10-27-user-directory-warning.md"